SpringData/src/main/java/com/epam/jmp2/controller/AccidentWebController.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jmp2.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.epam.jmp2.entities.Accident;
+import com.epam.jmp2.model.RoadAccident;
+import com.epam.jmp2.service.AccidentService;
+
+@Controller
+public class AccidentWebController {
+
+	@Autowired
+	private AccidentService service;
+
+	@RequestMapping("/list")
+	public String list(@RequestParam(value = "page", required = false, defaultValue = "0") int page, Model model) {
+		List<Accident> accidents = service.findAll(page);
+		model.addAttribute("accidentList", accidents);
```

Comment:
I had two controller: the AccidentController (of hometask 5) is annotated @RestController and serves restful web services, as to this controller, because home task asked for web pages (jsp) and if i just return list, i've got to display this result by javescript in list.jsp, so I use model here for the convenience of jsp <c:forEach> tags..